services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/model/QueryEnhancedRequest.java

```diff
@@ -121,6 +155,9 @@ public int hashCode() {
         return result;
     }
 
+    /**
+     * Use this builder to create a request with the desired parameters.
```

Comment:
Can we say what it is rather than just what you should do with it.

Reply:
Yes, updated.